### PR TITLE
[opt](compaction) Add logs for update delete bitmap info in cloud compaction

### PR DESCRIPTION
**Pull request by bobhan1**
## Proposed changes

add logs about update delete bitmap size in cloud compaction for debug for duplicate key in MOW table.

### CODE REVIEW

**Comment by bobhan1** [comment]
run buildall

File: be/src/cloud/cloud_base_compaction.cpp [hunk 1]
```diff
@@ -297,6 +297,14 @@ Status CloudBaseCompaction::modify_rowsets() {
...
                 _stats.merged_rows, initiator, output_rowset_delete_bitmap,
                 _allow_delete_in_cumu_compaction));
+        LOG_INFO("update delete bitmap in CloudBaseCompaction, tablet_id={}, range=[{}-{}]",
+                 _tablet->tablet_id(), _input_rowsets.front()->start_version(),
+                 _input_rowsets.back()->end_version())
+                .tag("job_id", _uuid)
+                .tag("input_rowsets", _input_rowsets.size())
+                .tag("input_rows", _input_row_num)
+                .tag("input_segments", _input_segments)
+                .tag("update_bitmap_size", output_rowset_delete_bitmap->delete_bitmap.size());
```
Comment (zhannngchen) [inline be/src/cloud/cloud_base_compaction.cpp:307]: should print the initiator id as well

Comment (zhannngchen) [reply be/src/cloud/cloud_base_compaction.cpp:307]: plz print the lock id for load job's update_delete_bitmap call

Comment (bobhan1) [reply be/src/cloud/cloud_base_compaction.cpp:308]: > plz print the lock id for load job's update_delete_bitmap call

lock_id is always -1 for compaction job

**Comment by bobhan1** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by bobhan1** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]

**Pull request merged**